packages/react/src/QuestionnaireForm/QuestionnaireForm.tsx

```diff
@@ -109,14 +126,15 @@ export function QuestionnaireForm(props: QuestionnaireFormProps): JSX.Element |
       }}
     >
       {questionnaire.title && <Title>{questionnaire.title}</Title>}
-
-      {questionnaire.item && (
+      {questionnaireItems && (
         <QuestionnaireFormItemArray
-          items={questionnaire.item}
+          items={questionnaireItems}
           answers={answers}
           onChange={setItems}
+          renderPages={numberOfPages > 1}
           activePage={activePage}
-          renderPages={numberOfPages > 0}
+          handleRepeatableItem={handleRepeatableItem}
```

Comment:
Event handler props should use `on` prefix, not `handle`.  By internal convention, we name our functions with the `handle` prefix (Mantine does this as well), but the props should always start with `on`.